roles/postgresql/templates/init.sh.j2

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+{% for user in postgresql_users %}
+psql -v ON_ERROR_STOP=1 --username "postgres" <<-EOSQL
+    CREATE USER {{ user.name }} WITH PASSWORD '{{ user.password }}';
+EOSQL
+{% endfor %}
```

Comment:
I was debating using [createuser](https://www.postgresql.org/docs/current/app-createuser.html) but reading the documentation I wonder how to properly deal with passing in the password.

Then my other thought was whether if it's better to create the users in a single query:
```suggestion
psql -v ON_ERROR_STOP=1 --username "postgres" <<-EOSQL
{% for user in postgresql_users %}
    CREATE USER {{ user.name }} WITH PASSWORD '{{ user.password }}';
{% endfor %}
EOSQL
```

And if you take that latter step to the extreme: should there just be a single script that creates the users and the databases.